content/en/network_monitoring/performance/network_page.md

```diff
@@ -60,19 +62,23 @@ Once the custom facet is created, use this tag to filter and aggregate traffic i
 ### Wildcard search
 To perform a multi-character wildcard search, use the `*` symbol as follows:
 
-- `service:web*` matches all services that start with web
-- `service:*web` matches all services that end with web
-- `service:*web*` matches all services that contain the string web
+- `client_service:web*` matches all client services that start with web
+- `client_service:*web` matches all client services that end with web
+- `client_service:*web*` matches all client services that contain the string web
 
-Wildcard searches work within facets with this syntax. This query returns all the services that end with the string mongo:
+Wildcard searches work within facets with this syntax. This query returns all the client services that end with the string mongo:
 
-`service:*mongo`
+`client_service:*mongo`
 
 To learn more, see the [search syntax][1] documentation.
 
+### Group by
+
+Groups allow you to filter your data by a given tag's value. For example, if you select a grouping such as **host**, results are grouped by individual hosts. You can also choose to not group by a tag using the **Ungrouped traffic** option. Additionally, you may have large chunks of data that are not tagged by the grouping you're interested in. In these situations, you can use **Auto-grouped traffic** to group data by whichever tags are available.
```

Comment:
```suggestion
Groups allow you to group your data by a given tag's value. For example, if you select a grouping such as **host**, results are grouped by individual hosts. You can also choose to view all your data in a single group using the **Ungrouped traffic** option. Additionally, you may have large chunks of data that are not tagged by the grouping you're interested in. In these situations, you can use **Auto-grouped traffic** to group data by whichever tags are available.
```

Reply:
Alternately, if you want to avoid repeating the word "group," the first line can be: "Groups allow you to aggregate your data..."